### PR TITLE
Validate the generated repo against their schema

### PR DESCRIPTION
**Pull request by jsancio**

### CODE REVIEW

**Comment by jsancio** [comment]
@larioj check it out.

File: scripts/validate-packages.py [hunk 1]
```diff
@@ -63,17 +64,27 @@ def _validate_revision(given_package, revision, path):
...
 
     # validate command.json
     command_json_path = os.path.join(path, 'command.json')
     command_json = None
     if os.path.isfile(command_json_path):
         eprint("\t\tcommand.json:", end='')
-        command_json = _validate_json(command_json_path, COMMAND_JSON_SCHEMA)
+        if packaging_version == "4.0":
+            sys.exit(
+                "\tERROR\n\nThis file is not support for version 4.0 packages"
```
Comment (larioj) [inline scripts/validate-packages.py]: Better phrasing: "This file is not supported in version 4.0 packages".

Comment (jsancio) [inline scripts/validate-packages.py]: Done.

**Comment by jsancio** [comment]
> Just one minor comment. How did you test this changes?

@larioj I manually ran validate-packages.py and gen-universe.py. Both "Mesosphere repo validation" and "ci" should run those scripts too.

**Pull request merged**